src/kernels/conv_3x3_wheel_alpha_v9_2_7.s

```diff
@@ -24,5 +24,9 @@
  *
  *******************************************************************************/
 .include "conv_3x3_wheel_alpha_v9_2_7_prologue.inc"
-.include "conv_3x3_wheel_alpha_v9_2_7.inc"
+.if (.amdgcn.gfx_generation_number == 8)
```

Comment:
CO V2 assember generates old predefined symbol, `.option.machine_version_major`, NOT `.amdgcn.gfx_generation_number`. So we are losing CO V2 compatibility.

I recommend including `rocm_version.inc`, that defines old symbols for CO V3, and using old symbols here.

[Note] Currently all ASM sources that support CO V2 use old `option...` symbols. Eventually we will change `rocm_version.inc` so it will define new symbols for CO V2 and use new `amdgcn...` symbols everywhere. If you wish, you can include this change right in this PR.

Reply:
Thank you for your careful reviewing, fixed in 4622fe5.
I also locally tested such cases and updated PR description.